contracts/src/integrations/cert/EigenDACertVerifier.sol

```diff
@@ -35,11 +35,28 @@ contract EigenDACertVerifier is
 
     DATypesV1.SecurityThresholds internal _securityThresholds;
 
+    // Each quorum that is part of this list MUST meet the confirmationThreshold defined in _securityThresholds
+    // in order for a cert to be considered valid.
+    // Recommended default value is 0x0001 to require the ETH and EIGEN quorums to sign.
     bytes internal _quorumNumbersRequired;
 
+    // This gas limit is passed to the checkDACertReverts call inside checkDACert.
+    // Any cert that causes checkDACertReverts to exceed this gas limit will be labeled with an INTERNAL_ERROR status code,
+    // because there is no way to distinguish out-of-gas errors from other low-level evm reverts in a try/catch...
+    // See https://rareskills.io/post/try-catch-solidity#gdvnie-9-what-gets-returned-during-an-out-of-gas?
+    // The main gas cost associated with verifying cert is the for loop that iterates on all non-signing operators.
+    //
+    // We recommend setting this to 15M gas for optimistic rollups that need the checkDACert call to be executed onchain
+    // in the one step prover contract, to allow for some buffer room for other gas costs in the one step prover tx.
+    // This is because Fusaka is introducing a 16,777,216 per-tx gas limit: https://eips.ethereum.org/EIPS/eip-7825
+    //
+    // For zk rollups and optimistic rollups with zk fraud proofs, this limit is less crucial, but still useful to enforce
+    // a max limit on the amount of prover compute units needs to validate a cert.
+    uint32 internal _checkDACertGasLimit;
```

Comment:
You may want to know that this particular variable may be set to immutable. What this would mean is that the variable would become part of the bytecode of the contract, rather than stored in contract storage. If this variable is not meant to be settable, it is probably better to label it as immutable.

Reply:
Oh I see that makes sense. Guessing `_quorumNumbersRequired` and `_securityThresholds` are not immutable because you can't have immutable bytes and structs?

Reply:
[17f0477](https://github.com/Layr-Labs/eigenda/pull/1953/commits/17f0477b9cc1b783196e1bc70d8bcae37c4e5826)